pkg/maps/eventsmap/eventsmap.go

```diff
@@ -71,7 +71,7 @@ func InitMap() error {
 		int(unsafe.Sizeof(Key{})),
 		&Value{},
 		int(unsafe.Sizeof(Value{})),
-		common.GetNumPossibleCPUs(log),
+		common.GetNumPresentCPUs(log),
```

Comment:
I don't know if we should use `present` or `possible` or what the role of 1256a13 is, but we want to use the same value in datapath and Golang for all maps. So we should also fix this for signalmap.

Reply:
We discussed this on previous PRs and deliberately chose `possible` so if there is any hotplugging, we would handle that correctly (ie not just drop events on newly hotplugged CPUs).

Do we actually need to be fixing up the BPF side to ensure that it opens using `possible` instead?

Reply:
@pchaigno @gandro @joestringer  It sounds like `-D__NR_CPUS__=$(nproc)` is broken then in `bpf_compile()` from bpf/init.sh. Does `echo $(nproc)` return 2 or 128?

Reply:
Agree on hotplug, if suddenly CPU 4 would start generating events these would otherwise not be visible if we use `present` instead of `possible`. All of BPF kernel side uses `possible`.

Reply:
@gandro reported that `nproc` returns 2. I think we want `nproc --all`. `runtime.NumCPU()` also returns 2.

Reply:
Yes, such that agent and datapath both has the `max_entries=128`.

Reply:
That sounds reasonable to me as well. I can try to whip up a fix and validate it. Are there cases where `__NR_CPUS__` should not be equal `nproc --all`?

Reply:
Alternative PR to have the datapath always use `present`: https://github.com/cilium/cilium/pull/12121